RELEASE.md

```diff
@@ -21,6 +21,8 @@
 * Updated `kedro pipeline create` and `kedro catalog create` to use new `/conf` file structure.
 * Converted `setup.py` in default template to `pyproject.toml` and moved flake8 configuration
   to dedicated file `.flake8`.
+* Fixed a bug that `OmegaConfigLoader` read config from hidden directory like `.ipynb_checkpoints`.
```

Comment:
This wasn't removed btw

Reply:
Thanks, I missed this when I merged from main and get conflicts